aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1014,14 +1014,18 @@ static Tensor _batch_tile_tensor(
 static Tensor _mask_to_indices(const Tensor& mask) {
   // This function returns a vector of the indices at which given
   // boolean mask is True. Here at::nonzero performs test (time/mem).
-  TORCH_CHECK(
-      mask.dim() == 1, "Currently _mask_to_indices only supports 1-d masks.");
+  TORCH_CHECK(mask.dim() == 1, "_mask_to_indices only supports 1-d masks.");
```

Comment:
Nit: We control the input here right , so I do wonder if these could be downgraded to debug assets.

Reply:
Do you mean I should replace this with TORCH_INTERNAL_ASSERT()? Or something else?

Reply:
@Skylion007 ?

Reply:
Yep @aartbik

Reply:
Specifically, TORCH_INTERNAL_ASSERT_DEBUG_ONLY but yeah it's a really minor nit

Reply:
Done